crates/compiler/test_mono/generated/lambda_set_niche_same_layout_different_constructor.txt

```diff
@@ -1,10 +1,6 @@
 procedure Test.1 (Test.4):
     ret Test.4;
 
-procedure Test.21 (Test.23, #Attr.12):
-    let Test.22 : Str = CallByName Test.5 Test.23 #Attr.12;
-    ret Test.22;
-
```

Comment:
these functions were previously added by the glue code, and now they are no longer